frontend/amundsen_application/static/js/pages/TableDetailPage/ReportTableIssue/styles.scss

```diff
@@ -28,8 +28,9 @@
     margin: 5px 0 0;
   }
 
-  .report-table-issue-buttons {
+  .submit-row {
     display: flex;
+    gap: 40px;
```

Comment:
I decided to change this to a relative percentage value instead of a specific px value, is this alright to leave in place, or is it best practice to pull that out to a variable as well?

Reply:
I would still make it into a variable, thanks!